cmake/std/PullRequestLinuxDriver.sh

```diff
@@ -217,7 +217,10 @@ cmake --version
 
 module list
 
-echo "MPI type = sems-${SEMS_MPI_NAME:?}/${SEMS_MPI_VERSION:?}"
+#This crashes for the serial case since the MPI variables are not set
+if [ "Trilinos_pullrequest_gcc_4.9.3_SERIAL" != "${JOB_BASE_NAME:?}" ] ; then
```

Comment:
Why not just search for `_SERIAL` in the job name?

Reply:
@mhoemmen I'm guessing is this is the easiest to do... If we add a 2nd SERIAL build in later then we'll probably fix that.

I did exactly the same thing Jim did on this too over on my PR branch, lol.  I think we must have seen the error at the same time :p